main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1813,23 +1814,6 @@ object Defaults extends BuildCommon {
       excludes: ScopedTaskable[FileFilter]
   ): Initialize[Task[Seq[File]]] = collectFiles(dirs: Taskable[Seq[File]], filter, excludes)
 
-  private[sbt] def configArtifactPathSetting(
```

Comment:
Why not keep the function but change the impl so we don't have to add mima exclusion?